public/app/plugins/panel/heatmap/types.ts

```diff
@@ -0,0 +1,21 @@
+export * from './panelcfg.gen';
+
+import { AxisPlacement, HeatmapCellLayout } from '@grafana/schema';
+
+import {
+  defaultPanelOptions as defaultPanelOptionsGen,
+  HeatmapColorMode,
+  HeatmapColorScale,
+  PanelOptions,
+} from './panelcfg.gen';
+
+defaultPanelOptionsGen.color!.mode = HeatmapColorMode.Scheme;
+defaultPanelOptionsGen.color!.scale = HeatmapColorScale.Exponential;
+defaultPanelOptionsGen.yAxis = {
+  axisPlacement: AxisPlacement.Left,
+};
+defaultPanelOptionsGen.rowsFrame = {
+  layout: HeatmapCellLayout.auto,
+};
+
+export const defaultPanelOptions = defaultPanelOptionsGen as PanelOptions;
```

Comment:
```suggestion
export const defaultPanelOptions = {
  ...defaultPanelOptionsGen,
  color: { ...defaultPanelOptionsGen.color, mode: HeatmapColorMode.Scheme, scale: HeatmapColorScale.Exponential },
  yAxis: { ...defaultPanelOptionsGen.yAxis, axisPlacement: AxisPlacement.Left },
  rowsFrame: { ...defaultPanelOptionsGen.rowsFrame, layout: HeatmapCellLayout.auto },
} as PanelOptions;
```
This can be simplified a bit.